### DISCUSSION TITLE
Howto set temperature via MQTT

### DISCUSSION THREAD

**Discussion by klaus-weindel** [slvwolf/panasonic-comfort-cloud-mqtt#23] [General]
I get:
homeassistant/climate/pcc_eg-wohnzimmer_ac/state {"mode": "heat", "power": "on", "target_temperature": 20.5, "inside_temperature": 22, "outside_temperature": 17, "update_epoch": 1711635753.9048176}

but how can I set target_temperature?

**Comment by slvwolf** [comment]
Setting the temperature would be by sending the value, for example `23`, to
`homeassistant/climate/pcc_eg-wohnzimmer_ac/temp_cmd`

Other topics for controlling the device can be found from the discovery event,
(https://github.com/slvwolf/panasonic-comfort-cloud-mqtt/blob/main/pcfmqtt/events.py#L60)

If using Home Assistant with auto discovery enabled the device should be automatically added to the instance.
